client/my-sites/upgrades/domain-management/components/domain/maintenance-card.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react' ;
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { getSelectedDomain } from 'lib/domains';
+import Notice from 'components/notice';
+
+const MaintenanceCard = ( { domains, selectedDomainName, translate } ) => {
+	const domain = getSelectedDomain( { domains, selectedDomainName } ),
+		lastIndexOfDot = domain.name.lastIndexOf( '.' ),
+		tld = lastIndexOfDot !== -1 && domain.name.substring( lastIndexOfDot );
```

Comment:
I admit to knowing absolutely nothing about this or how to test it but is it at all possible that a user would have the domain `test-again.co.uk` leading to the tld being shown as `.uk`?